pallets/parachain-staking/src/lib.rs

```diff
@@ -234,8 +234,7 @@ pub mod pallet {
 		Delegations<AccountId, Balance>
 	{
 		pub fn sort_greatest_to_least(&mut self) {
-			self.delegations
-				.sort_unstable_by(|a, b| b.amount.cmp(&a.amount));
+			self.delegations.sort_by(|a, b| b.amount.cmp(&a.amount));
```

Comment:
:+1: 

Let me know if we need fresh benchmarks...

Reply:
We do, I'll run them tonight